src/index.js

```diff
@@ -1,17 +1,10 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
-import './index.css';
 import App from './App';
-import reportWebVitals from './reportWebVitals';
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
   <React.StrictMode>
     <App />
   </React.StrictMode>,
 );
-
-// If you want to start measuring performance in your app, pass a function
-// to log results (for example: reportWebVitals(console.log))
-// or send to an analytics endpoint. Learn more: https://bit.ly/CRA-vitals
-reportWebVitals();
```

Comment:
- [ ] As mentioned in the project requirements all the boilerplate that got created by the CRA (*texts*, *images*, and *styles*) should be deleted which means all the files and images in the public directory except the `index.html` file, the index.css, the reportWebVitals.js and setupTests.js files, App.test.js, and the logo.svg 🙃 🙃